src/internal/storage/chunk/storage.go

```diff
@@ -62,7 +62,8 @@ func NewStorage(objC obj.Client, memCache kv.GetPut, db *pachsql.DB, tracker tra
 // NewReader creates a new Reader.
 func (s *Storage) NewReader(ctx context.Context, dataRefs []*DataRef, opts ...ReaderOption) *Reader {
 	client := NewClient(s.store, s.db, s.tracker, nil)
-	return newReader(ctx, client, s.memCache, s.deduper, s.prefetchLimit, dataRefs, opts...)
+	defaultOpts := []ReaderOption{WithPrefetchLimit(s.prefetchLimit)}
```

Comment:
Shouldn't the defaults be what we initialize the reader with?  Maybe we define them in a constant, and then we refer to the constant when we create the `reader` in the structure.

Reply:
Not sure I follow. The default mentioned here is the default based on the storage configuration, not the default for the reader. A storage instance can be created with a default prefetch limit that will be used for readers. This can be overridden by clients for particular use cases with the functional options provided to `NewReader`.